### ISSUE TITLE
Tab unsuspends immediately after suspending

### ISSUE THREAD

**Issue by mbecroft** [deanoemcke/thegreatsuspender#1246]
Extension version: 7.1.6

Browser: Tested on both:
- Chrome v 85.0.4183.102 (Official Build) (64-bit)
- Chrome v 83.0.4103.116 (Official Build) (64-bit)

Operating system: GNU/Linux

When I have a tab open, and choose to suspend it, it suspends and I briefly see the suspended-tab screen, but it then immediately unsuspends.

This behaviour began directly after the latest Great Suspender update was released (I think in the last 1-2 days). I have never encountered this issue before.

**Comment by mbecroft** [comment]
Needless to say I've tried every combination of settings to see if it helps with this! I do have "Apply Chrome's built-in memory-saving when suspending" enabled, but I'm not sure what the consequences are of turning it off while I have dozens of already-suspended tabs.

**Comment by tunetopj** [comment]
What I noticed is the tabs are "Automatically unsuspending when it is viewed"
 There is a setting related to that which I have turned off but tabs are unsuspending nevertheless.

**Comment by mbecroft** [comment]
@tunetopj This is exactly the problem I am having. That said - on one of my systems I just upgraded chrome to the latest release, and ended up with a new release 7.1.8 of The Great Suspender.

After upgrading and restarting Chrome, the problem has gone away. I don't know if it was the upgrade or the restart that made the difference.

So I think with confirmation that it is also OK now for others, this ticket could be closed.

**Comment by Jaxenormus** [comment]
If you were able to figure out what was causing your issue and it was resolved, you can close this issue

**Comment by mbecroft** [comment]
I don't know what caused it, but I do know that some combination of possibly a further update and/or updating Chrome and/or restarting Chrome solved the issue.

One thing I'd note is on a Chrome instance with several hundred tabs, mostly suspended - the most recent update that was pushed out did not go smoothly. In the Chrome task manager, Suspender was now taking up over 2 GB RAM, and Chrome was slow and hardly responding. It was only after restarting Chrome that things went back to normal. So, maybe there were just challenges with the live, in-place upgrade of the latest Great Suspender, perhaps only in cases of ridiculously large numbers of tabs.

I'd invite feedback from @tunetopj whether his problem has also been resolved - but also, there may be lessons learned for smooth in-place upgrades in future.

**Comment by XxX-Force** [comment]
For those who are not aware, there is great concern that this extension may now be malicious in nature.
You will find a great deal of discussion about this here:

https://github.com/greatsuspender/thegreatsuspender/issues/1175

and a summary of the situation to date here:

https://github.com/greatsuspender/thegreatsuspender/issues/1263

**Comment by Jaxenormus** [comment]
No need to copy and paste the same message over and over again there are already issues addressing this

**Comment by XxX-Force** [comment]
@CalebDelbridge .. I was pointing people **to** those issues. There are clearly people (and now there are less) creating issues who were not aware of this situation, and if my pasting this warning into about a dozen posts is something that bothers you, then I suggest that you block me , report me, or both. I'd much rather make an attempt to help other users stay safe, and at least give them access to information they may not have been aware of, even if it only helps one person (as happened here: https://github.com/greatsuspender/thegreatsuspender/issues/1248), then worry about some GitHub etiquette. In any event, it was obviously over a day and a half ago when I stopped posting the few warnings that I did.

**Comment by ossilator** [comment]
you still reached only about a dozen additional people, while two million more are still in the dark. so that's clearly not the way to go, even if we assume that your "mission" is a legitimate one. please delete your comments from the issues where no followup to them was posted.

**Comment by XxX-Force** [comment]
@ossilator You have no idea how many people were reached by those messages. Not everyone who reads a message posts a reply. Certainly did more to help reach people than anything you've done. I'm not deleting anything. I made those posts days ago. They aren't hurting anyone. Get over it and go find something actually productive to do, instead of harassing me for actually trying to help people. I see a new issue [posted here](https://github.com/greatsuspender/thegreatsuspender/issues/1264). It's patently obvious that this person has no idea what's going on. Perhaps you'd like to tell them? Or shall I make yet another post for you to pointlessly complain about?

**Comment by ossilator** [comment]
you'll still reach only 0.01% of the users at best, so what you're doing is _ineffective_, and it's annoying those who try to use the tracker productively. having good intentions is unfortunately not enough to actually do good.

**Comment by XxX-Force** [comment]
@ossilator **YOU** are literally the one causing people to still get notifications regarding this, and creating the annoyance, by repeatedly whining about posts that were made **days** ago.

>"..._what you're doing is ineffective_..."

I'm not "**_DOING_**" anything at all. For whatever reason, you seem to be struggling with the concept of the linear nature of time. I made a few posts in the _past_, and have not made any of those posts since.

I'm not going to argue about your philosophy of "_it's only good to do good if the amount of good you're doing is arbitrarily good enough_".

Two days ago I made maybe a dozen posts in the most recently submitted issues in order to help bring awareness to the situation. That's it. As far as I'm concerned, they will stay there until the Earth is engulfed by the Sun, as I have zero intention of removing those few innocuous posts. IF **YOU** could somehow find the self-control to stop trying to impose your will upon others, and stop repeatedly dragging this pointless nonsense back up over and over (and over) again, then it would not be the exact problem that you are complaining about, and that you are literally the one creating, all by yourself, right now.

**Comment by ossilator** [comment]
you made it rather clear that you believe that what you did was correct, so discussing this further is very much in order, as it is about your possible future actions. it's unfortunate that three other people have to witness this, but that's the situation you created.
your condescension is duly noted, but won't do anything to advance your point.
to sum the situation up, you have self-righteously violated the etiquette in order to advance an agenda that is essentially futile (because this is the wrong forum) and misguided to start with (because TGS isn't any more concerning than thousands of other closed source extensions at this point, give or take some obvious sloppiness). that you're clearly not getting the social hints and are instead resorting to right-wing/libertarian talking points about imposed wills (with the implication that this is not appropriate for the situation) doesn't speak well of your judgment.

**Comment by XxX-Force** [comment]
@ossilator Every time you reply to me, you create more of the same problem you are complaining about. Since you apparently lack the self-control to stop, I will now no longer respond to your comments. You have completely and needlessly blown this situation WAY out of proportion, throwing in wildly speculative and baseless assumptions (a habit of yours as can be seen here:https://github.com/greatsuspender/thegreatsuspender/issues/1175#issuecomment-721071506), 

I honestly don't know how to respond to this "right-wing libertarian" remark. Not only is it completely wrong, (it's none of your business, but I'm a left-leaning progressive democrat), inappropriate and irrelevant to the discussion, it's also just weird. I think you may want to talk to someone. You make connections where none exist. You exhibit paranoia when just some healthy skepticism would be fine. You are talking about actions that I'm doing in the present tense, when they occurred in the past, while at the same time talking about my "_possible future actions_". The only reason ANYONE is witnessing this is because you just wont STFU about it. How about you go find a 5 minute meditation video and take a nice long deep breath? In with the good. Out with the bad. Aahhhh.

Look. I made a few posts. About a dozen. Two Days Ago. They were simple, short posts containing exactly 41 words each. 214 characters over 8 lines. Posts made in good faith, to warn people who might not be aware of the potentially dangerous security concerns regarding the extension they are using. They weren't part of any '_ADVANCING AN AGENDA_' or '_MY MAYBE SUSPICIOUS NOT QUITE LEGITIMATE "MISSION"_'. It was just me, trying to do my part and hopefully reach (and help) a few people.

That's it. I am completely done explaining it (again), because you're correct.. I am not "_advancing my point_" with you no matter what I say. Which frankly is fine. It would have been nice if you were able to come around to see the innocent nature of this ridiculously minor situation, but my world will keep on spinning just perfectly without your approval.

To sum up the situation, You note my condescension while talking about my 'agendas' & 'missions', my right-wing libertarianism, my ineffectiveness in trying to help people (if that's even what I'm _really_ trying to do), my self-righteousness, my lack of good judgment ... OK, that's enough for me. Please get off of my back and go find someone else to bother, or find a fly to pull wings off of, or whatever else you do for fun. Regardless, discussing this further is very much over.

**Comment by ossilator** [comment]
it's somewhat unfortunate that this discussion is generating some pointless traffic, but it's nothing compared to you flooding 16 other issues, and i'm fairly sure that it's negligible in the context of the surrounding shit-show. i'm more concerned about the distraction your misplaced contributions will cause to later readers, and the impact of possible later "interventions" of yours if you don't learn anything from this episode.

you really should pay more attention to what is being said, to avoid all that misquoting, straw-manning, and generally overreacting. your post also appears to be full of weapons-grade projection, which would be funny if it wasn't so sad. take a step back, man.
